pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -723,6 +733,7 @@
     </xs:complexType>
   </xs:element>
 
+  <!-- vertex_count, triangle_count, bounding_box_min and bounding_box_max should not be here, they can be extracted from the geometry -->
   <xs:element name="terrain">
```

Comment:
`vertex_count`, `triangle_count`, `bounding_box_min` and `bounding_box_max` should not be here, they can be extracted from the geometry

Reply:
This was discussed some time ago, the general sense is that these elements are helpful for software that will process a PVCollada file. I agree that they can be derived from the content, so perhaps these elements can be removed after some usage experience.